root/static/scripts/tests/autocomplete2.js

```diff
@@ -66,7 +66,7 @@ $(function () {
             entityType={entityType}
             id="entity-test"
             key={entityType + '-autocomplete'}
-            onChange={console.log}
+            onChange={console.log} // eslint-disable-line react/jsx-handler-names
```

Comment:
Small nitpick, but this (and the one below) are a bit over the max-len constraint of 78 characters. I'd rather put them above the line with eslint-disable-next-line instead, if you don't mind :)

Reply:
Good spot! Fixed with https://github.com/metabrainz/musicbrainz-server/pull/1292/commits/40f49b0647c06281103c254cfaadaaee0e2bba44